pom.xml

```diff
@@ -148,7 +148,7 @@
     <vers.hbase2>0.98.3-hadoop2</vers.hbase2>
     <vers.hbase-cdh4>0.94.6-cdh${cdh4.version}</vers.hbase-cdh4>
     <vers.hbase-cdh5>0.96.1.1-cdh${cdh5.version}</vers.hbase-cdh5>
-    <vers.hive>0.10.0</vers.hive>
+    <vers.hive>0.13.0</vers.hive>
```

Comment:
0.13.0 is the first version with the CHAR and VARCHAR types. I thought support for those was significant, plus we need to keep up with the upstream libraries better. We already test with 0.12.0-cdh5, so this isn't a huge jump in terms of compatibility.